lib/scala/project-manager/src/test/scala/org/enso/projectmanager/protocol/ProjectCreateDefaultToLatestSpec.scala

```diff
@@ -1,10 +1,18 @@
 package org.enso.projectmanager.protocol
 
 import io.circe.literal.JsonStringContext
+import nl.gn0s1s.bump.SemVer
 import org.enso.projectmanager.BaseServerSpec
+import org.enso.runtimeversionmanager.CurrentVersion
 
 class ProjectCreateDefaultToLatestSpec extends BaseServerSpec {
+
+  private val testVersion = SemVer("0.1.0").get
+
   "project/create" should {
+
+    CurrentVersion.internalOverrideVersion(testVersion)
```

Comment:
I think that it may be worth un-doing this override after the tests are executed.

Maybe even worth adding a helper method:
```
withOverriddenVersion(testVersion: SemVer)(action: => Unit): Unit = {
  val original = CurrentVersion.version
  CurrentVersion.internalOverrideVersion(testVersion)
  try { action }
  finally {
    CurrentVersion.internalOverrideVersion(original)
  }
}
```

Because I'm wondering if this change could spill and affect other tests - better to avoid risking that.